src/ResourceManager/IotHub/Commands.IotHub/help/Get-AzureRmIotHubCertificate.md

```diff
@@ -0,0 +1,151 @@
+---
+external help file: Microsoft.Azure.Commands.IotHub.dll-Help.xml
+Module Name: AzureRM.IotHub
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRmIotHubCertificate
+
+## SYNOPSIS
+Lists all certificates or a particular certificate contained within an Azure IoT Hub. 
+
+## SYNTAX
+
+### ResourceSet
+```
+Get-AzureRmIotHubCertificate [-ResourceGroupName] <String> [-Name] <String> [[-CertificateName] <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceIdSet
+```
+Get-AzureRmIotHubCertificate -ResourceId <String> [[-CertificateName] <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+For a detailed explanation of CA certificates in Azure IoT Hub, see https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-x509ca-overview
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Get-AzureRmIotHubCertificate -ResourceGroupName "myresourcegroup" -Name "myiothub"
+
+ResourceGroupName	Name		CertificateName Status     Expiry
+-----------------	----        ---------------	------     ------
+myresourcegroup		myiothub3	mycert1         Unverified 12/04/2027 13:12
```

Comment:
fix formatting here

Reply:
this is only showing in git, as discussed ignoring this comment